### PR TITLE
[Console] Fixed SymfonyStyle when get terminal width fails

### PR DESCRIPTION
**Pull request by 1ed**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Thanks! Can you please amend the patch reported by fabbot also?

**Comment by ogizanagi** [comment]
@1ed : Thanks ! :+1:

**Comment by 1ed** [comment]
Done

**Comment by fabpot** [comment]
:+1:

File: src/Symfony/Component/Console/Style/SymfonyStyle.php [hunk 1]
```diff
@@ -316,8 +316,8 @@ private function getProgressBar()
     private function getTerminalWidth()
     {
         $application = new Application();
-        list ($width, $height) = $application->getTerminalDimensions();
+        list($width, $height) = $application->getTerminalDimensions();
```
Comment (stloyd) [inline src/Symfony/Component/Console/Style/SymfonyStyle.php]: `$height` is never used...

Comment (1ed) [inline src/Symfony/Component/Console/Style/SymfonyStyle.php]: :+1: :smiley:

**Comment by sstok** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @1ed.

**Pull request merged**